src/org/zaproxy/zap/extension/alertFilters/resources/help_fil_PH/contents/alertFilter.html

```diff
@@ -2,54 +2,49 @@
+ipinapakita ng panel na ito ang isang listahan ng lahat ng alerto ng mga filter na kasama ang mga buton para sa pag dadagdag, pag-alis at pagtanggal sa kanila.
 </p>
 <p>
-To use this add-on you will need to:
+Para magamit ang add-on na ito kailangan ninyong:
 <ul>
-<li>Add your application to a context</li>
-<li>Open the Session Properties dialog (eg by double clicking on the context)</li>
-<li>Select the contexts' 'Alert filters' panel</li>
-<li>Click 'Add' and fill in the details for each alert you want to override</li>
+<li>Ang iyong aplikasyon ay idagdag sa isang konteksto</li>
+<li>Buksan ang Sesyon ng mga Pag aari ng diyalogo (eg sa dobleng pag click sa konteksto)</li>
+<li>Piliin ang mga konteksto' 'Alerto ng mga Filter' panel</li>
+<li>I-click ang 'add' at punan ang mga detalye sa bawat alerto na gusto mong i-override</li>
 </ul>
-Note: The alert filters will only be applied to new alerts, not to any alerts that were
-already present when the filter was created. 
+Paalala: Ang alerto ng mga filter ay mailalagay lamang sa bagong mga alerto, hindi sa kahit na anong mga alerto na naroroon kapag ang filter ay nagawa na. 
 </p>
 <p>
-The 'add' and 'modify' dialogs have the following fields: 
+Ang 'add' at 'modify'na diyalogo ay mayroong sumusunod na mga field.
 
-<h3>Alert</h3>
-A pull down containing all of the active and passive alert rules currently installed.
-
-<h3>New Risk Level</h3>
-The new risk level to be assigned to any alerts raised that match the criteria defined by the rule.
+<h3>Bagong Lebel ng Panganib<0>
+ang bagong lebel ng panganib na itinalaga sa kahit na anong mga alerto na itinaas na magiging kapareho ng pamantayan sa tinutukoy na patakaran.
  
 <h3>URL</h3>
-An optional URL.<br>
-If specified then this rule will be applied if the URL matches the URL of a raised alert.<br>
+ang isang opsyonal na URL.<br>
+kung tinakda at ang isang URL ay tiniyak saka ita-trato ang URL bilang isang regex ekspresiyon pagkatapos ay ihahambing sa URL ng mga alerto.<br>
+
+<h3>Ang URL ay Regex?</h3>. 
+kung tinakda at ang isang URL ay tiniyak saka ita-trato ang URL bilang isang regex ekspresiyon pagkatapos ay ihahambing sa URL ng mga alerto.<br>
+kung hindi naitakda pagkatapos ay alin man sa tiniyak na URL ay dapat na eksaktong magkahambing sa URL ng mga alerto.
 
-<h3>URL is Regex?</h3>
-If set and a URL is specified then the URL will be treated as a regex expression when compared with the URL of the alert.<br>
-If it is not set then any specified URL must exactly match the URL of the alert.
+<h3>Parametro<h3>
+Ang isang opsyonal na parametro.<br>
+kung tinukoy at pagkatapos ay nailapat ang patakaran na ito kung ang parametro ay eksaktong magkakatulad sa parametro ng itinaas na alerto.<br>
 
-<h3>Parameter</h3>
-An optional parameter.<br>
-If specified then this rule will be applied if the parameter exactly matches the parameter of a raised alert.<br>
+<0>pinagana</h3>
```

Comment:
Grrr.....again.

Reply:
Addressed. Crowdin could just hide the tags (instead of adding placeholders).

Reply:
Thanks!